pages/Useful Utilities/Screen-Sharing.md

```diff
@@ -1,13 +1,13 @@
 Screensharing is done through PipeWire on Wayland.
 
-## Prereqs
+# Prerequsities
 
 Make sure you have `pipewire` and `wireplumber` installed, enabled and running
 if you don't have them yet.
 
 Also install `slurp` for monitor selection.
 
-## Screensharing
+# Screensharing
 
```

Comment:
We could benefit from adding a section about patching wlroots for screensharing to work on nvidia.

Reply:
Relevant patch to mention: https://aur.archlinux.org/cgit/aur.git/plain/0001-nvidia-format-workaround.patch?h=hyprland-nvidia-screenshare-git

Reply:
@fufexan the gist by powerball *already* mentions the patch. And you'll also notice that the screenshare package is maintained by the same person.

Reply:
Ok, thanks.